mautic/mautic
# Adding marketplace command

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [y]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR adds a new section to the Markeplace's plugin detail page where the users can copy the command to install the plugin:
![Screenshot 2023-06-28 at 16 47 30](https://github.com/mautic/mautic/assets/1235442/419b0bf4-47b9-43f6-ab43-79922f3ed327)


I also noticed that if the installation fails via UI then it sometimes never returns any error response. So that's fixed too.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to the Market place
3. Click on some plugin
4. Check that the command section is there.
5. Copy the command and execute it via CLI
6. It should install the plugin. Your Instance will be broken most probably because the plugin is for the M4 version. But that's an issue for another day. Just delete it from the plugins directory and clear cache.
